src/components/Breadcrumbs/index.tsx

```diff
@@ -66,10 +74,10 @@ const BreadcrumbFragment: React.VFC<BreadcrumbFragmentProps> = ({
   );
 };
 
-const Container = styled.div`
+const Container = styled.nav`
   display: flex;
   align-items: center;
-  margin-bottom: ${size.l};
+  margin-bottom: ${size.s};
```

Comment:
The larger margin is somewhat necessary due to the headers being squashed when the browser window is smaller 😖

<img width="676" alt="Screen Shot 2022-12-02 at 12 06 24 PM" src="https://user-images.githubusercontent.com/47064971/205346533-0107095d-a1d6-442f-b171-781c6175b016.png">